src/main/java/Calculator.java

```diff
@@ -0,0 +1,73 @@
+// It's 1949 and you are tasked to lead programmer's team in order to implement a brand new calculator. The calculator is very simple though: it can add (+), multiple(x), divide(%) and take(-). It does not and will not support any brackets or memory. It will only ever support whole numbers.
+// Calculator should have only 1 function- int calculate(string expression)
+
+public class Calculator {
+
+    private char[] defaultOperators = {'+', '-', 'x', '%', '/', '!'};
+    private char[] charArray;
+
+
+    public int calculate (String expression) {
+
+        // empty string should return 0
+        if (expression == "" || expression.isEmpty()){
+            return 0;
+        }
+        // 3 - Division from 0
+        // Division from zero should not be possible. If a string "5/0" is passed, result should be -1
+        if (expression.contains("/0")) {
+            return -1;
```

Comment:
You could use `private static final float INVALID_DEFAULT = -1;`

Reply:
Apply in all places instead of -1. -1 might change in the future with some other value, but like so, you will be able to control the **change to be kept in a single place**